README.md

```diff
@@ -1,81 +1,70 @@
-## Synchronisation & Encryption
-#### Synchronisation
-The sync process consists in having two actors, one is the ClientServer, which exists on a Virtual Machine and the second one being the CentralServer.
+## Synchronisation and Encryption
 
-##### Setup ClientServer
-The fallowing script needs to be run on TAO instance in order to make a client server.
+### Synchronisation
+The synchronization process is based upon two actors, the Client Server, which is installed on a Virtual Machine, and the Central Server.
```

Comment:
OK, I'm really nitpicking here, but the title of the paragraph is written in a British manner, while here a `z` suddenly appears in `synchronization`.
Let's pick one language variety and stick to it.

Reply:
To be honest, I find that very reasonable, considering that I'm the one who always insists on good grammar and spelling. Good catch!